.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -3,6 +3,7 @@ name: Bug Report
 description: Report a bug
 labels:
   - status/triage
+  - kind/bug
```

Comment:
I think we originally had this, but then removed it? cc @tonistiigi 

I remember a discussion (I think in the build team internal docker slack) about wanting to ensure that all issues with the label were *confirmed* bugs, which requires a level of triage before.

Reply:
👋 @jedevc - I had a chance to read the old thread you mentioned, and the scenario described is crucial (being able to find confirmed bugs).

The goal with this was to try and help pre-sort as the issue entered into triage and be able to identify when the issue had been responded/handled in any way. In this case, a response could include requesting more information, putting the issue into a "blocked" state but no longer requiring initial sorting. This idea stemmed from status/triage being the default state for all inbound bugs and feature requests _and_ an assumption that the first response would get it out of that triage state to something else to avoid recycling issues. Would the addition of a verified (or using the existing accepted label) still address your use case? 

I am hoping for simplicity above all else on this front - any and all ideas would be amazing 😄

Reply:
`kind/maybe-bug` until confirmed? 😁

Reply:
SGTM :tada: Essentially anything that helps filtering!

Reply:
Added in the `kind/maybe-bug` as well as a "bug report lite" for reports that might be a little light on details out the gate.